src/components/settings/SettingsAppearanceTab/ThemeCard.jsx

```diff
@@ -1,40 +1,65 @@
 import PropTypes from 'prop-types';
-import { Card, Form } from '@edx/paragon';
+import { Button, Card, Form } from '@edx/paragon';
 import ThemeSvg from './ThemeSvg';
+import { SCHOLAR_THEME } from '../data/constants';
+
+export default function ThemeCard({
+  themeVars, selected, setTheme, openCustom,
+}) {
+  const isCustom = themeVars.title === 'Custom Theme';
+  function selectTheme(vars) {
+    setTheme([vars, selected[1]]);
+  }
+
+  function deleteCustom() {
+    if (selected[0].title === 'Custom Theme') {
+      setTheme([SCHOLAR_THEME, null]);
+    } else {
+      setTheme([selected[0], null]);
+    }
+  }
+
+  // AFTER DELETE MUST CODE
 
-export default function ThemeCard({ themeVars, selected, setTheme }) {
   return (
     <Card style={{ width: '18rem' }} isClickable>
       <Card.Section className="p-0">
         <ThemeSvg themeVars={themeVars} />
       </Card.Section>
-      <Card.Footer className="justify-content-between pt-2.5">
+      <Card.Footer className="justify-content-between pt-2.5 pb-0">
         <h3>{themeVars.title}</h3>
         <Form.Radio
-          checked={selected.title === themeVars.title}
-          onChange={() => setTheme(themeVars)}
+          checked={selected[0].title === themeVars.title}
+          onChange={() => selectTheme(themeVars)}
           data-testid={`radio-${themeVars.title}`}
           name="theme"
-          defaultChecked={themeVars.title === 'Scholar (Default)'}
           value={themeVars.title}
         />
       </Card.Footer>
+      {isCustom && (
+      <div className="ml-auto">
+        <Button className="" variant="tertiary" onClick={() => deleteCustom()}>Delete</Button>
+        <Button className="" variant="tertiary" onClick={() => openCustom()}>Edit</Button>
+      </div>
+      )}
+      {!isCustom && (<div className="pt-3" />)}
```

Comment:
nit: would it be worth conditionally adding a `pb-3` class to the above element when `isCustom` is false rather than rendering a new `div`? E.g., `className={classNames({ 'pb-3': !isCustom })}`